py3status/modules/group.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
+"""
+Group a bunch of modules together and switch between them.
+
+In `i3status.conf` groups can be configured. The active one of these groups is
+shown in the i3bar.  The active group can be changed by a user click.  If the
+click is not used by the group module then it will be passed down to the
+displayed module.
+
+Modules can be i3status core modules or py3status modules.
+
+Additionally the active group can be cycled through automatically.
+
+Configuration parameters:
+    button_next: Button that when clicked will switch to display next module.
+        Setting to `0` will disable this action. (default 4)
+    button_prev: Button that when clicked will switch to display previous
+        module.  Setting to `0` will disable this action. (default 5)
+    color: If the active module does not supply a color use this if set.
+        Format as hex value eg `'#0000FF'` (default None)
+    cycle: Time in seconds till changing to next module to display.
+        Setting to `0` will disable cycling. (default 0)
+    fixed_width: Reduce the size changes when switching to new group
+        (default True)
+    format: Format for module output. (default "GROUP: {output}")
+
+
+Format of status string placeholders:
+    {output} Output of current active module
+
+Example:
+
+```
+# Create a disks group that will show space on '/' and '/home'
+# Change between disk modules every 30 seconds
+...
+order += "group disks"
+...
+
+group disks {
+    cycle = 30
+    format = "Disks: {output}"
+
+    disk "/" {
+        format = "/ %avail"
+    }
+
+    disk "/home" {
+        format = "/home %avail"
+    }
+}
+```
+
+@author tobes
+"""
+
+from time import time
+
+
+class Py3status:
+    # available configuration parameters
+    button_next = 4
+    button_prev = 5
+    color = None
+    cycle = 0
+    fixed_width = True
+    format = "GROUP: {output}"
+
+    class Meta:
+        include_py3_module = True
+
+    def __init__(self):
+        self.items = []
+        self.active = 0
+        self._cycle_time = time() + self.cycle
```

Comment:
this is not a good place to do such a thing because the module is instantiated **before** the user's configuration parameters are applied, this means that the self.cycle will always be 0.

the result here is that the first displayed value will actually be the second module listed in the group :(